woudc_data_registry/search.py

```diff
@@ -413,6 +413,61 @@
                 'fields': {'raw': typedefs['keyword']}
             }
         }
+    },
+    'contributions': {
```

Comment:
Perfect! Remember if you change any fields (i.e. please remove `instrument_id` and maybe `instrument_model`) to change this as well.